### PR TITLE
Benji/photofixes

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/Flows/Onboarding/OnboardingCoordinator.swift [hunk 1]
```diff
@@ -28,10 +28,7 @@ class OnboardingCoordinator: PresentableCoordinator<Void> {
          deepLink: DeepLinkable?) {
 
         self.reservationId = reservationId
-        self.reservationCreatorId = reservationCreatorId
-
-        //self.reservationId = "3fx7TOiQey"//reservationId
-        //self.reservationCreatorId = "FZr3kzCeT9" //reservationCreatorId
+        self.reservationCreatorId = reservationCreatorId ?? "IQgIBSPHpE"
```
Comment (Benjamindodgson) [inline Shared/Flows/Onboarding/OnboardingCoordinator.swift:31]: This is a backup admin user.

File: Shared/Flows/Onboarding/OnboardingViewController.swift [hunk 2]
```diff
@@ -142,6 +142,10 @@ class OnboardingViewController: SwitchableContentViewController<OnboardingConten
         self.photoVC.onDidComplete = { [unowned self] result in
             switch result {
             case .success:
+                Task {
+                    try await ActivateUser().makeRequest(andUpdate: [], viewsToIgnore: [self.view])
```
Comment (Benjamindodgson) [inline Shared/Flows/Onboarding/OnboardingViewController.swift:146]: Moving this to onboarding so its not called every time we take a photo.

File: Shared/Flows/Onboarding/PhotoViewController.swift [hunk 3]
```diff
@@ -33,14 +33,10 @@ class PhotoViewController: ViewController, Sizeable, Completable {
...
-    private let avatarView = AvatarView()
-
-    private let beginButton = Button()
-    private let confirmButton = Button()
-    private let retakeButton = Button()
-
-    private let buttonContainer = View()
-    private var buttonContainerRect: CGRect?
+    private let button = Button()
+    private let instructionLabel = Label(font: .smallBold, textColor: .background4)
```
Comment (Benjamindodgson) [inline Shared/Flows/Onboarding/PhotoViewController.swift:37]: A bunch of better UI for taking a photo. Less buttons, easier UX.

File: iOS/Flows/Archives/ArchiveCoordinator.swift [hunk 4]
```diff
@@ -42,12 +42,19 @@ class ArchiveCoordinator: PresentableCoordinator<Void> {
         ToastScheduler.shared.delegate = self
 
         self.archiveVC.addButton.didSelect { [unowned self] in
+
+ //           self.presentPhoto()
```
Comment (Benjamindodgson) [inline iOS/Flows/Archives/ArchiveCoordinator.swift:46]: Leaving this in so I can do more testing on the next issue.

File: iOS/Flows/Archives/ArchiveCoordinator.swift [hunk 5]
```diff
@@ -164,3 +171,20 @@ extension ArchiveCoordinator: ArchiveViewControllerDelegate {
         self.router.present(coordinator, source: self.archiveVC)
     }
 }
+
+private class ProfilePhotoViewController: PhotoViewController {
```
Comment (Benjamindodgson) [inline iOS/Flows/Archives/ArchiveCoordinator.swift:175]: Same here.

**Review by MartinJibber — APPROVED** [review]
Coolio

**Pull request merged**